jsx/Form.js

```diff
@@ -1385,6 +1385,7 @@ class ButtonElement extends React.Component {
         <div className={this.props.columnSize}>
           <button
             type={this.props.type}
+            name={this.props.name}
```

Comment:
@driusan is this change okay to be in this PR?

Reply:
This PR will likely conflict with the changes made to `ButtonElement` from PR #3907. Although, discussions regarding those changes still need to be resolved.